src/common/containers/repository-setup.js

```diff
@@ -73,24 +77,20 @@ RepositorySetup.propTypes = {
 };
 
 const mapStateToProps = (state, ownProps) => {
-  const account = ownProps.params.account.toLowerCase();
-  const repo = ownProps.params.repo.toLowerCase();
-  const fullName = `${account}/${repo}`;
+  const owner = ownProps.params.owner.toLowerCase();
+  const name = ownProps.params.name.toLowerCase();
+  const fullName = `${owner}/${name}`;
+  const url = getGitHubRepoUrl(fullName);
```

Comment:
Third place where we repeat the same as in build details and builds listing (see comment https://github.com/canonical-ols/build.snapcraft.io/pull/91/files#r96604386)